src/index.js

```diff
@@ -0,0 +1,190 @@
+const state = { temp: '', currentCity: defaultCity };
+
+const currentTemp = document.getElementById('current-temp');
+const upButton = document.getElementById('up-temp');
+const downButton = document.getElementById('down-temp');
+
+const currentLandscape = document.getElementById('landscape-weather-visual');
+const currentSky = document.getElementById('sky-weather-visual');
+const skySelector = document.getElementById('sky-select');
+
+const currentCityDisplay = document.getElementById('city');
+const cityInput = document.getElementById('city-input');
+const resetCityButton = document.getElementById('city-reset');
+
+const realTimeButton = document.getElementById('real-time-button');
+
+const dateTime = document.getElementById('date-time');
+const currentDate = new Date().toUTCString();
+dateTime.textContent = currentDate.slice(0, 3) + ' ' + currentDate.slice(5, 16);
+
+const select = document.querySelector('select');
+
+cityInput.placeholder = defaultCity;
+currentCityDisplay.textContent = state.currentCity;
+
+const setUpTemp = () => {
+  state.temp += 1;
+  displayTemp();
+  changeColor();
+  changeLandscape();
+  autoChangeSky();
+};
+
+const setDownTemp = () => {
+  state.temp -= 1;
+  displayTemp();
+  changeColor();
+  changeLandscape();
+  autoChangeSky();
+};
+
+const changeColor = () => {
+  if (state.temp < 50) {
+    currentTemp.setAttribute('class', 'teal');
+    //(currentTemp.style.color = 'teal')?
+  } else if (state.temp < 60) {
+    currentTemp.setAttribute('class', 'green');
+  } else if (state.temp < 70) {
+    currentTemp.setAttribute('class', 'yellow');
+  } else if (state.temp < 80) {
+    currentTemp.setAttribute('class', 'orange');
+  } else if (state.temp >= 80) {
+    currentTemp.setAttribute('class', 'red');
+  }
+};
+
+const changeLandscape = () => {
+  if (state.temp < 60) {
+    currentLandscape.textContent = '🍂 🌲🌲 ⛄️ 🌲 ⛄️ 🍂🌲 ⛄️ 🍂 🌲';
+  } else if (state.temp < 70) {
+    currentLandscape.textContent = '🌾 🌾 🍃 🪨 🌾_🛤_🌾 🌾 🌾 🍃 ';
+  } else if (state.temp < 80) {
+    currentLandscape.textContent = '🌸 🌿 🌼 ☘️ 🌷 🌻 ☘️ 🌸 🌱 🌻 🌷';
+  } else if (state.temp >= 80) {
+    currentLandscape.textContent = '🌵 _🐍 _🦂_ 🌵 _🐍 _🏜 _🦂';
+  }
+};
+
+const updateCityName = city => {
+  if (city === '') {
+    currentCityDisplay.textContent = defaultCity;
+    select.value = 'default';
+  } else {
+    state.currentCity = city;
+    currentCityDisplay.textContent = inputCaseSensitive(city);
+    select.value = 'default';
+  }
+};
+
+const resetCity = () => {
+  updateCityName(defaultCity);
+  cityInput.value = '';
+};
+
+const apiCalls = () => {
+  let lat, lon;
+  axios
+    .get('http://127.0.0.1:5000/location', {
+      params: {
+        q: currentCityDisplay.textContent,
+      },
+    })
+    .then(response => {
+      lat = response.data[0].lat;
+      lon = response.data[0].lon;
+
+      findWeather(lat, lon);
+    })
+    .catch(error => {
+      console.log('could not retrieve location', error.data);
+    });
+};
```

Comment:
Nice `axios.get`! Small nit: this function has a very generic name, maybe something like `findLocation` would work better (and match the other API function on L105)!